.travis/install.sh

```diff
@@ -11,11 +11,11 @@ then
 	 PYTHON_COMMAND=python3
 	 PIP_COMMAND=pip3
 fi
- 
-$PIP_COMMAND install -r requirements.txt
+
 $PYTHON_COMMAND setup.py install
 $PYTHON_COMMAND -m art test
 $PYTHON_COMMAND -m art test2
+$PIP_COMMAND install -e .[Coverage]
```

Comment:
What is this whole commit for?
Can you please explain the reason?

Reply:
First I added this line to test `art` library in develop mode, but later removed it ;-)